src/cdm.yml

```diff
@@ -0,0 +1,26 @@
+---
+- hosts: all
+  name: Configure for the CISA CDM environment
+  vars:
+    ports:
+      # CrowdStrike
+      - direction: out
+        port: 443
+        proto: tcp
```

Comment:
Did we want to reposition this to be more alphabetically sorted?

Reply:
Please see commit 4ccac31.

Reply:
Please see commit 4ccac31.